x-pack/filebeat/module/mysqlenterprise/audit/test/mysql_audit_test.log-expected.json

```diff
@@ -0,0 +1,1100 @@
+[
+    {
+        "@timestamp": "2020-10-01T19:21:33.000Z",
+        "event.action": "mysql-startup",
+        "event.category": [
+            "database"
+        ],
+        "event.dataset": "mysqlenterprise.audit",
+        "event.kind": "event",
+        "event.module": "mysqlenterprise",
+        "event.outcome": "unknown",
+        "fileset.name": "audit",
+        "host.os.full": "x86_64-Linux",
+        "input.type": "log",
+        "log.offset": 0,
+        "mysqlenterprise.audit.class": "audit",
+        "mysqlenterprise.audit.connection_id": 0,
```

Comment:
do we know what it means if you have a `connection_id` field at startup? Just wondering if we should try and figure out an ECS mapping for it or not

Reply:
I don't think there is any. At first I tried to find a "session ID" field we could use in ECS; but it seems that the session ID is only unique to the current active connections, so it will reset and use the same connection_id for another connection later according to its docs:

https://dev.mysql.com/doc/refman/8.0/en/information-functions.html#function_connection-id

Do you know any other field that might match? I tried to look in client, user and network but could not find anything that matches well